easybuild/easyconfigs/m/MapSplice/MapSplice-2.2.1-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'MapSplice'
+version = '2.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.netlab.uky.edu/p/bioinfo/%(name)s2'
+description = """MapSplice is a software for mapping RNA-seq data to reference genome for splice
+ junction discovery that depends only on reference genome, and not on any further annotations."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://protocols.netlab.uky.edu/~zeng']
+sources = ['%(name)s-v%(version)s.zip']
+
+patches = [
+    '%(name)s-%(version)s_fix-build-gcc.patch',
+]
+
+checksums = [
+    '4f3c1cb49ba0abcfc952de5946ee0b56db28c51f4f4d4f5abca66b4461ca7d05',  # MapSplice-v2.2.1.zip
+    '30c29dfde280b9614b4624224eed0927243acb45f99554e898de29df4c967a29',  # MapSplice-2.2.1_fix-build-gcc.patch
+]
+
+dependencies = [
+    ('Python', '2.7.12'),
+]
+
+files_to_copy = ['bin', 'mapsplice.py', 'Copyright.txt']
+
+sanity_check_paths = {
+    'files': ['%(namelower)s.py'],
+    'dirs': ['bin'],
```

Comment:
@SethosII Since binaries are being built, I think it makes sense to specifically check for a couple of them (the main ones, if it's easy to tell which ones those are), rather than just checking for a non-empty `bin` directory.

Reply:
@boegel The `bin` directory contains a lot of binaries and I don't really know the main ones, but probably `bowtie` and `samtools`? Here is the list of binaries:

```
AddFusionStrandConsistent
alignment_handler
alignment_handler_multi
bowtie
bowtie-build
bowtie-inspect
bsb4
check_index_consistency
check_reads_format
cluster
collectstats
comp_fusiondb_offset
Convert2FusionAlignment
DNA2StdRegion
filter_1hits
FilterFusionAlignmentsByFilteredFusions
FilterFusionByNormalPaired
filterjuncbyROCarguNonCanonical
filteroriginalfusion
filterremappedfusion
find_mate_sam_fq
fusionsam2junc_filteranchor_newfmt
generate_combined_sequence
generate_fusiongene_convert_coordinate_trim
generate_fusiongene_convert_coordinate_trim_dRanger
gtf2genetab
junc2bed
junc_db
junc_db_fusion
load_fusion_chrom_seq_std
mapsplice_multi_thread
matchfusion2normal
MPSSam2fq
newsam2junc
parseCluster
read_chromo_size
reads2unmappedsam
recover_fusion_alignments_order
RemovePairNo
sam2fq
samtools
search_fusion_gene
search_unmapped_reads
SeparateNormalFromFusionJunc
sepdRangerfusion
sepMPSfusion
SepSam
SepSamUnmapped
SetUnmappedBitFlag
swap_dRanger_and_MPS_matched
```

Reply:
In that case, let's leave it as it...